src/shippingservice/Dockerfile

```diff
@@ -42,6 +42,7 @@ EXPOSE 50051
 ENTRYPOINT ["/src/shippingservice"]
 
 FROM without-grpc-health-probe-bin
+# renovate: datasource=github-releases depName=grpc-ecosystem/grpc-health-probe
```

Comment:
Praise:
This is cool (i.e., that we can [reference GitHub repo releases](https://docs.renovatebot.com/modules/datasource/#github-releases-datasource))!